recipes/xcube-geodb/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: xcube_geodb
+  version: {{ version }}
+
+source:
+  url: https://github.com/dcs4cop/xcube-geodb/archive/0.1.0.dev0.tar.gz
+  sha256: ad67c4f6a53ea482b2ce738a3cd77df5d1d8679bcc50291756d6e7e7ae9fd226
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+  entry_points:
+    - geodb = xcube_geodb.cli.main:main
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - pip
+    - xcube >=0.3.0
+    - geopandas >=0.5
+    - matplotlib
+    - descartes
+    - requests
+    - sphinx
+    - python-dotenv
+    - jupyterlab
+    - psycopg2
+
+test:
+  imports:
+    - xcube_geodb.core.geodb
+
+about:
+  summary: "geodb is a Python package"
+  home: https://github.com/dcs4cop/xcube-geodb
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  description: "geodb is a Python package"
+  doc_url: https://xcube_geodb.readthedocs.io
+  dev_url: https://github.com/dcs4cop/xcube-geodb
+
+extra:
+  recipe-maintainers:
+    - dzelge
+    - forman
```

Comment:
@forman are you OK with being a maintainer?

Reply:
@scopatz sure, thanks!